main.js

```diff
@@ -0,0 +1,236 @@
+const CurrentWeather = function () {
+
+  // Property name to be saved in local storage when a user picks a default location
+  const STORAGE_ID = 'Weather';
```

Comment:
Usually, you want these to be less generic and more specific because in a larger application, you don't want to have conflicts across views.  I'd name it something like 'weather-homepage-default-location'.